iohk-monitoring/src/Cardano/BM/Output/Monitoring.lhs

```diff
@@ -149,25 +168,88 @@ spawnDispatcher mqueue config sbtrace = do
                                    $ HM.toList ls
 \end{code}
 
+\begin{code}
+getVarValuesForMonitoring :: LoggerName
+                          -> Configuration
+                          -> [(LoggerName, LogObject a)]
+                          -> IO [(VarName, Measurable)]
+getVarValuesForMonitoring logName config mbuf =
+    HM.lookup logName <$> getMonitors config >>= \case
+        Just (_, mEvExpr, _) -> do
+            let varNames = extractVarNames mEvExpr
+            return . catMaybes . concat $ map (getVNnVal varNames) mbuf
+        Nothing -> return []
+  where
+    extractVarNames expr = case expr of
+        Compare vn _  -> [vn]
+        AND     e1 e2 -> extractVarNames e1 ++ extractVarNames e2
+        OR      e1 e2 -> extractVarNames e1 ++ extractVarNames e2
+        NOT     e     -> extractVarNames e
+
+    getVNnVal varNames logObj = case logObj of
+        (_, LogObject _ _ (LogValue vn val))       -> if vn `elem` varNames
+                                                      then [Just (vn, val)]
+                                                      else []
+        (_, LogObject _ _ (AggregatedMessage agg)) -> concat $ map getMeasurable agg
+        (_, _)                                     -> []
+      where
+        getMeasurable :: (Text, Aggregated) -> [Maybe (VarName, Measurable)]
+        getMeasurable agg = case agg of
+            (vn, AggregatedEWMA (EWMA _ val)) -> if vn `elem` varNames
+                                                 then [Just (vn <> ".ewma.avg", val)]
+                                                 else []
+            (vn, AggregatedStats st)          -> if vn `elem` varNames
+                                                 then stValues vn st
+                                                 else []
+            _                                 -> []
+          where
+            stValues vn st =
+                [ Just (vn <> ".flast", flast st)
+                , Just (vn <> ".fold",  fold st)
+
+                , Just (vn <> ".fbasic.fmin",   fmin   . fbasic $ st)
+                , Just (vn <> ".fbasic.fmax",   fmax   . fbasic $ st)
+                , Just (vn <> ".fbasic.fcount", PureI . fromIntegral . fcount . fbasic $ st)
+                , Just (vn <> ".fbasic.fsum_A", PureD                . fsum_A . fbasic $ st)
```

Comment:
access to the fields is great!
but, we do not need to provide access to `fsum_A` and `fsum_B` as they are just internally used to calculate "mean" and "stdev". can you add:
".fbasic.mean", PureD . meanOfStats . fbasic $ st
".fbasic.stdev", PureD . stdevOfStats . fbasic $ st
(or similar..)